docs/guides/vault/address-restrict-to-vault.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_label: 'Restrict Addresses to Vaults'
+sidebar_position: 4
+---
+
+# Restrict Addresses to Vaults
+
+As mentioned previously, the **Vault** can be used to restrict different addresses (protocols, other devices, etc..) to execute and set Data on it instead of setting and executing directly on the Universal Profile. 
```

Comment:
```suggestion
As mentioned previously, the **Vault** can be used to restrict different addresses (protocols, other devices, etc..) to execute and set data on it instead of setting and executing directly on the Universal Profile. 
```